exampleappandroidcompose/src/main/java/com/example/exampleappandroidcompose/MainActivity.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.exampleappandroidcompose
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
```

Comment:
I believe we don't need many of the imports here. Please use the `optimize imports` tool. I also recommend to enable it in the Android Studios' settings.
![image](https://github.com/rootstrap/FlowForms/assets/26490822/c733b2c8-c671-4ba4-a6e7-487c98693298)